bbb-export-annotations/package.json

```
@@ -18,5 +18,9 @@
   "devDependencies": {
     "eslint": "^8.20.0",
     "eslint-config-google": "^0.14.0"
+  },
+  "engines": {
+    "node": "^16.16.0",
+    "npm": "^8.11.0"
```

```suggestion
    "npm": "^8.5.0"
```

This has been working correctly with npm 8.5.0 and this is the version our packaging is set to work with right now. Technically I should upgrade our packaging https://gitlab.senfcall.de/senfcall-public/docker-bbb-build/-/blob/main/Dockerfile#L148 but I recall there were some hiccups last time which made me stick to 8.5.0. Are you fine tweaking this to the 8.5.0 version for the moment (it works well with it)?